src/utils/hmac-validator.ts

```diff
@@ -35,24 +31,19 @@ export function generateLocalHmac({
     ...(host && {host}),
   });
   return crypto
-    .createHmac('sha256', config.apiSecretKey)
+    .createHmac('sha256', apiSecretKey)
     .update(queryString)
     .digest('hex');
 }
 
-/**
- * Uses the received query to validate the contained hmac value against the rest of the query content.
- *
- * @param query HTTP Request Query, containing the information to be validated.
- */
-export default function validateHmac(query: AuthQuery): boolean {
+export function validateHmac(apiSecretKey: string, query: AuthQuery): boolean {
```

Comment:
Putting a note here that this is changed in another branch, so these values can be obtained from config, when `validateHmac` is attached using a `create__(config)` type function.